app/views/admin/accounts/show.html.haml

```diff
@@ -1,8 +1,14 @@
 - content_for :page_title do
   = @account.acct
 
+- proofs = @account.identity_proofs.active
+- fields = @account.fields
```

Comment:
need?

Reply:
It's not needed. Sorry, this was leftover from a previous attempt at that PR!
Removed it